### PR TITLE
Functionality/cards stats chart

### PR DESCRIPTION
**Pull request by michaelputnam67**
# Description drawing

- chart js
- testing for chart

## Issues

Closes # 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser

# Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

**Review by mrlandino — APPROVED** [review]
Looks good.

File: src/Components/stats-chart/statsChart.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import './statsChart.scss'
```
Comment (userigorgithub) [inline src/Components/stats-chart/statsChart.tsx:1]: Nice usage of chart feature.

**Review by friotious — APPROVED** [review]
Looks good.  Want to see chart !

**Review by userigorgithub — APPROVED** [review]
LGTM (looks good to me).

**Pull request merged**